react/src/components/masthead/SprkMasthead.stories.js

```diff
@@ -4,6 +4,24 @@ import SprkButton from '../buttons/SprkButton';
 import SprkLink from '../../base/links/SprkLink';
 import SprkDropdown from '../dropdown/SprkDropdown';
 
+export default {
+  title: 'Components|Masthead',
+  component: SprkMasthead,
+  decorators: [
+    story => <div>{story()}</div>
```

Comment:
What does the masthead need this <div> decorator for?

Reply:
left this question on Friday @RVMendoza

Reply:
I figured they all needed them, which sets them to be available to change if it ever comes up.

I can remove it.